### ISSUE TITLE
Report secondary clusters in text format

### ISSUE THREAD

**Issue by fungs** [MrOlm/drep#24]
Hi, thanks for the useful program. I wonder if you can get the secondary clusters in text format other than a list of pairwise similarity scores. Basically, I have to redo the clustering, or extract the data from the images. Both seems wrong to me.

Best,
Johannes

**Comment by MrOlm** [comment]
Hi Johannes,

I’m not sure what you mean about having the clustering information in “text
form”? There is a table called “Cdb.csv” which lists all of the genomes and
which cluster they’re in. Is this what you’re looking for?

Best,
-Matt

On Mar 9, 2018, at 7:00 AM, Johannes Dröge <notifications@github.com> wrote:

Hi, thanks for the useful program. I wonder if you can get the secondary
clusters in text format other than a list of pairwise similarity scores.
Basically, I have to redo the clustering, or extract the data from the
images. Both seems wrong to me.

Best,
Johannes

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/MrOlm/drep/issues/24>, or mute the thread

**Comment by fungs** [comment]
Hi Mat,

true, I just didn't see that it also contains the secondary cluster assignments. Thank you, this is just what I was looking for.

Best,
Johannes
